src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,59 @@
+package duke.ui;
+
+import java.util.Scanner;
+
+public class Ui {
+    /* Rem bot logo */
+    public final String LOGO = " ___  ___  __ __ \n"
+            + "| . \\| __>|  \\  \\\n"
+            + "|   /| _> |     |\n"
+            + "|_\\_\\|___>|_|_|_|\n";
+
+    /* line filler */
```

Comment:
Just a heads up, single line JavaDocs should still start with /**.

Reply:
Ah yes, thanks!